src/check-rates/index.html

```diff
@@ -20,7 +20,7 @@
 
           <h2 class="page-title">Explore interest rates<sup class="beta">BETA</sup></h2>
 
-          <p class="lead">Our data comes from actual lenders and is updated every day. Credit score, loan type, home price, and down payment amount all affect the interest rate you can get. Interest is only one of the many costs you will pay when getting a mortgage. While shopping, ask about <a href="http://www.consumerfinance.gov/askcfpb/136/what-are-discount-points-or-points.html" target="_blank">points</a>, <a href="http://www.consumerfinance.gov/askcfpb/1953/what-is-mortgage-insurance-and-how-does-it-work.html" target="_blank">mortgage insurance</a>, and <a href="http://www.consumerfinance.gov/askcfpb/139/what-are-closing-costs.html" target="_blank">closing costs</a>. Make a final decision only after comparing lenders’ <a href="http://www.consumerfinance.gov/askcfpb/146/what-is-a-good-faith-estimate-what-is-a-gfe.html" target="_blank">Good Faith Estimates</a> and <a href="http://www.consumerfinance.gov/askcfpb/180/what-is-a-final-truth-in-lending-disclosure.html" target="_blank">Truth in Lending disclosures</a>, which include all the costs.</p>
+          <p class="lead">Our data comes from actual lenders and is updated every day. Credit score, loan type, home price, and down payment amount all affect the interest rate you can get. Interest is only one of the many costs you will pay when getting a mortgage. While shopping, ask about <a href="http://www.consumerfinance.gov/askcfpb/136/what-are-discount-points-or-points.html" target="_blank">points</a>, <a href="http://www.consumerfinance.gov/askcfpb/1953/what-is-mortgage-insurance-and-how-does-it-work.html" target="_blank">mortgage insurance</a>, and <a href="http://www.consumerfinance.gov/askcfpb/139/what-are-closing-costs.html" target="_blank">closing costs</a>. Make a final decision only after comparing lenders’ <a href="http://consumerfinance.gov/askcfpb/1995/what-is-a-loan-estimate.html" target="_blank">Loan Estimates</a> which include all the costs.</p>
```

Comment:
Not sure about the rest of the updated links but you should double check if there are affected tests.